assets/component-product-grid.css

```diff
@@ -1,21 +1,21 @@
 .product-grid .grid__item {
-  padding-bottom: 2rem;
+  padding-bottom: calc(2rem + var(--page-width-margin));
 }
 
 .product-grid.negative-margin {
-  margin-bottom: -2rem;
+  margin-bottom: calc(-2rem - var(--page-width-margin));
 }
 
 @media screen and (min-width: 750px) {
   .product-grid .grid__item {
-    padding-bottom: 5rem;
+    padding-bottom: calc(5rem + var(--page-width-margin));
   }
 
   .product-grid.negative-margin {
-    margin-bottom: -5rem;
+    margin-bottom: calc(-5rem - var(--page-width-margin));
   }
 
   .product-grid.negative-margin--small {
-    margin-bottom: -1rem;
+    margin-bottom: calc(-3rem + var(--page-width-margin));
```

Comment:
this was calling for `-1rem` originally. Is it meant to be calling for `-3` instead ?

Reply:
good catch @ludoboludo